opennms-doc/guide-user/src/asciidoc/text/alarms/alarm-sound-flash.adoc

```diff
@@ -0,0 +1,84 @@
+
+:imagesdir: ../../images
+
+[[gu-alarm-sounds]]
+=== Alarm Sounds
+
+Often users want an audible indication of a change in alarm state. 
+The _{opennms-product-name}_  alarm list page has the optional ability to 
+generate a sound either on each new alarm or (more annoyingly) 
+ on each change to an alarm event count on the page. 
+
+The figure <<gu-alarm-sounds-ui, Alarm Sounds View>> shows the alarm list page when alarms sounds are enabled.
+
+[[gu-alarm-sounds-ui]]
+.Alarm Sounds View
+image:alarms/AlarmSound.jpg[]
+ 
+By default the alarm sound feature is disabled. System Administrators must activate the
+ sound feature and also set the default sound setting for all users. 
+However users can modify the default sound setting for the duration of their logged-in 
+session using a drop down menu with the following options:
+
+* Sound off: no sounds generated by the page.
+* Sound on new alarm: sounds generated for every new alarm on the page.
+* Sound on new alarm count: sounds generated for every increase in alarm event count for alarms on the page.
+
+=== Flashing Unacknowledged Alarms 
+
+By default _{opennms-product-name}_  displays the alarm list page with acknowledged and unacknowledged alarms
+listed in separate search tabs. In a number of operational environments it is useful to see all of the 
+alarms on the same page with unacknowledged alarms flashing to indicate that they haven't yet been 
+noticed by one of the team. This allows everyone to see at a glance the real time status of all alarms 
+and which alarms still need attention.
+
+The figure <<gu-alarm-sounds-ui, Alarm Sounds View>> also shows the alarm list page when flashing unacknowledged alarms
+are enabled. Alarms which are unacknowledged flash steadily. Alarms which have been acknowledged do not flash and
+also have a small tick beside the selection check box. All alarms can be selected to be escalated, cleared, 
+acknowledged and unacknowledged.
+
+=== Configuring Alarm Sounds and Flashing
+
+By default _{opennms-product-name}_ does not enable alarm sounds or flashing alarms. The system administrator must
+enable these features by changing the following settings in 
```

Comment:
I would advise the administrator to create a file like `opennms.properties.d/alarm.flashing.properties` and overwrite the default with custom settings. This would make it easier on updates and don't change the `opennms.properties` directly.